bin/rector

```diff
@@ -30,6 +31,8 @@ $autoloadIncluder->includeDependencyOrRepositoryVendorAutoloadIfExists();
 $autoloadIncluder->autoloadFromCommandLine();
 
 try {
+    checkMinimumPHPVersion();
```

Comment:
Better use class, functions are very hard to autoload and re-use. Hell to maintain and decouple